cvpy/image/Image.py

```diff
@@ -272,3 +273,35 @@ def get_image_array_const_ctype(image_binaries, dimensions, resolutions, ctype,
         num_cells = np.prod(resolution)
 
         return Image.get_image_array_from_row(image_binaries[n], dimension, resolution, ctype, channel_count)
+
+    @staticmethod
+    def convert_wide_to_numpy(wide_image) -> np.ndarray:
+
+        ''''
+        Convert a wide image to a numpy image array.
+
+        Parameters:
+        ----------
+        wide_image: bytes
+             Specifies the wide byte image array
```

Comment:
wide_image: bytes => wide_image: bytes buffer

Specifies the wide byte image array => Specifies the wide image byte buffer